easybuild/easyconfigs/b/BerkeleyGW/BerkeleyGW-1.0.6-intel-2016.02-GCC-4.9.eb

```diff
@@ -0,0 +1,35 @@
+easyblock = 'MakeCp'
+
+name = 'BerkeleyGW'
+version = "1.0.6"
+
+homepage = 'http://www.berkeleygw.org'
+description = """The BerkeleyGW Package is a set of computer codes that calculates the quasiparticle
+                 properties and the optical responses of a large variety of materials from bulk
+                 periodic crystals to nanostructures such as slabs, wires and molecules."""
+
+toolchain = {'name': 'intel', 'version': '2016.02-GCC-4.9'}
+toolchainopts = {'usempi': True}
+
+source_urls = ['http://www.berkeleygw.org/releases/']
+sources = ['BGW-%(version)s.tar.gz']
+checksums = ['8740562da41e87a62eb8d457675e4bfd']
+
+patches = ['BerkeleyGW_ignore_archmk.patch']
+
+files_to_copy = ['*']
+
+prebuildopts = 'COMPFLAG=-DINTEL PARAFLAG=-DMPI MATHFLAG=-DUSESCALAPACK FCPP="cpp -ansi" F90free="$MPIF90 -free" '
+prebuildopts += 'LINK="$MPIF90" FOPTS="$FFLAGS" MOD_OPT="-module " INCFLAG="-I" C_PARAFLAG=-DPARA CC_COMP="$MPICXX" '
+prebuildopts += 'C_COMP="$MPICC" C_LINK="$MPICXX" C_OPTS="$CFLAGS" REMOVE="/bin/rm -f" MKLPATH="$MKLROOT" '
+prebuildopts += 'FFTWLIB="$MKLROOT/lib/intel64/libfftw2xf_double_intel.a" FFTWINCLUDE="$FFTW_INC_DIR" '
```

Comment:
regarding the environment variables set by easybuild, yes, I've been doing that, or using the logs, just wondered if there was a better way to interactively test/debug

regarding fftw3, in version 1.1-beta2 of BGW, one could use FFTW 3 instead of 2 (by adding -DFFTW3), but one would still need the Fortran wrapper (libfftw3xf...), those are for C